src/sender.rs

```diff
@@ -752,10 +757,10 @@ where
     }
 
     /// Send `Keys` synchronization message
-    #[tracing::instrument(skip(self))]
+    #[tracing::instrument(skip(self), fields(recipient = recipient.service_id_string()))]
```

Comment:
```suggestion
    #[tracing::instrument(skip(self), fields(recipient = %recipient.service_id_string()))]
```

Using the display version is less verbose and more readable in my experience.